src/response.rs

```diff
@@ -54,7 +80,39 @@ impl Response {
         }
     }
 
-    /// Get the `StatusCode`.
+    /// Get the `StatusCode` of this `Response`.
+    ///
+    /// ```rust
+    /// # extern crate reqwest;
+    /// # #[macro_use] extern crate error_chain;
+    /// # use reqwest::{Client, StatusCode};
+    /// # use reqwest::header::Location;
+    /// #
+    /// # error_chain! {
+    /// #     foreign_links {
+    /// #         Reqwest(reqwest::Error);
+    /// #     }
+    /// # }
+    /// #
+    /// # fn run() -> Result<()> {
+    /// let client = Client::new()?;
+    /// let resp = client.post("http://httpbin.org/post")?.send()?;
+    /// match *resp.status() {
+    ///     StatusCode::Ok => println!("All good!"),
+    ///     StatusCode::Found => {
```

Comment:
reqwest will follow redirects in most cases. What if we show a very common thing to do: check if the code was a 404 (`StatusCode::NotFound`)?

If we wanted to include multiple examples, I could imagine some others too:

1. Check for `resp.status() == StatusCode::Ok`, `println!("response is not ok")` if not.
2. Make use of the `is_*` methods of a `StatusCode`, to be more flexible. `if resp.status().is_success()`, or `if resp.status().is_server_error()`, etc.
3. If someone needed to inspect for a very specific status code, like `StatusCode::PayloadTooLarge` (413), when trying to upload a file.

Would showing examples in that order, from most common to more niche/advanced, be useful?